lib/knife-zero/devpatch/train_connector.rb

```diff
@@ -0,0 +1,26 @@
+require 'chef/knife/bootstrap/train_connector'
+require 'knife-zero/helper'
+
+class Chef
+  class Knife
+    class Bootstrap < Knife
+      class TrainConnector
+        class_eval do
+          def connect!
+            # Force connection to establish
+            connection.wait_until_ready
+            if connection.is_a? Train::Transports::SSH::Connection
+              connection
+              .instance_variable_get(:@session)
```

Comment:
__[RuboCop]__ Layout/MultilineMethodCallIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines. ([link](https://sider.review/gh/repos/23179315/pulls/125#issue-4171799))

<sub>posted by Sider</sub>